tetris.py

```diff
@@ -489,5 +468,28 @@ def timer(self):
             self.topbar_time.config(text=f"{int(game_time / 60):02d}:{int(game_time % 60):02d}")
             self.topbar_time.after(1000, self.timer)
 
+    def new_game(self, event):
```

Comment:
This can be called with event (from `.bind("<f>" ...)`) or without event (from `self.new_game()`):

```suggestion
    def new_game(self, event=None):
```